cpp/benchmarks/pairwise_linestring_distance.cu

```diff
@@ -77,7 +77,7 @@ std::tuple<rmm::device_vector<cartesian_2d<T>>, rmm::device_vector<int32_t>> gen
   std::vector<cartesian_2d<T>> points(num_points);
 
   auto random_walk_func = [segment_length](auto const& prev, auto const& rad) {
-    return cartesian_2d<T>{prev.x + segment_length * rad.x, prev.y + segment_length * rad.y};
+    return cartesian_2d<T>{{prev.x + segment_length * rad.x, prev.y + segment_length * rad.y}};
```

Comment:
```suggestion
    return cartesian_2d<T>({prev.x + segment_length * rad.x, prev.y + segment_length * rad.y});
```

Reply:
It's mysterious that we have three different builds taking place, one that works, one that works with your fix, and one that works with mine.